File: 백준/Bronze/32684. 장기/장기.go
package main

import (
	"fmt"
)

func main() {


	var choki, eungyu [6]float64
	for i := 0; i < 6; i++ {
		fmt.Scan(&choki[i])
	}
	for i := 0; i < 6; i++ {
		fmt.Scan(&eungyu[i])
	}

	var cocjr_score float64 = (choki[0] * 13) +
		(choki[1] * 7) +
		(choki[2] * 5) +
		(choki[3] * 3) +
		(choki[4] * 3) +
		(choki[5] * 2)

	var eungyu_score float64 = (eungyu[0] * 13) +
		(eungyu[1] * 7) +
		(eungyu[2] * 5) +
		(eungyu[3] * 3) +
		(eungyu[4] * 3) +
		(eungyu[5] * 2) + 1.5


	if cocjr_score < eungyu_score{
		fmt.Println("ekwoo")

	}else{
		fmt.Println("cocjr0208")
	}
}

File: 백준/Bronze/32684. 장기/장기.rs
use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Write};

fn main() {
    let mut reader = BufReader::new(stdin().lock());
    let mut writer = BufWriter::new(stdout().lock());

    let mut input = reader.lines();

    if let Some(Ok(line)) = input.next() {
        let cocjr = line
            .trim()
            .split_whitespace()
            .map(|x| x.parse().unwrap())
            .collect::<Vec<usize>>();
        if let Some(Ok(line)) = input.next() {
            let ekwoo = line
                .trim()
                .split_whitespace()
                .map(|x| x.parse().unwrap())
                .collect::<Vec<f32>>();

            let cocjr_score = cocjr[0] * 13
                + cocjr[1] * 7
                + cocjr[2] * 5
                + cocjr[3] * 3
                + cocjr[4] * 3
                + cocjr[5] * 2;

            let ekwoo_score = ekwoo[0] * 13.
                + ekwoo[1] * 7.
                + ekwoo[2] * 5.
                + ekwoo[3] * 3.
                + ekwoo[4]* 3.
                + ekwoo[5] * 2.
                + 1.5;

            if cocjr_score as f32 > ekwoo_score {
                writeln!(writer, "{}", "cocjr0208").unwrap();
            } else {
                writeln!(writer, "{}", "ekwoo").unwrap();
            }
        }
    }
    writer.flush().unwrap();
}
